packages/client/lib/commands/CLIENT_KILL.spec.ts

```diff
@@ -65,6 +65,16 @@ describe('CLIENT KILL', () => {
             );
         });
 
+        it('MANAGE', () => {
```

Comment:
do you mean MAXAGE?

Reply:
I think I didn't finish fixing this one, as was unsure it was neccesary for v4 anymore.  Should i fix v4?